Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu
+LABEL maintainer="Francesco Di Natale dinatale3@llnl.gov"
+
+RUN apt-get update
+RUN apt-get install -y python python-pip git
```

Comment:
Docker recommends defining `apt-get update` and `apt-get install` as part of the same `RUN` statement to avoid caching issues: https://docs.docker.com/develop/develop-images/dockerfile_best-practices